examples/Extensions/RemoveEntireSitelinkCampaignExtensionSetting.php

```diff
@@ -0,0 +1,274 @@
+        $extensionFeedItemResourceNames = self::getAllSitelinkExtensionFeedItems(
+            $googleAdsServiceClient,
+            $customerId,
+            $campaignId
+        );
+
+        // Creates mutate operations, each of which contains an extension feed item operation
+        // to remove the specified extension feed items.
+        $mutateOperations = array_merge(
+            $mutateOperations,
+            self::createExtensionFeedItemMutateOperations($extensionFeedItemResourceNames)
+        );
+
+        // Issues a mutate request to remove the campaign extension setting and its extension
+        // feed items.
+        $response = $googleAdsServiceClient->mutate($customerId, $mutateOperations);
+        $mutateOperationResponses = $response->getMutateOperationResponses();
+
+        // Prints the information on the removed campaign extension setting and its extension feed
+        // items.
+        // Each mutate operation response is returned in the same order as we passed its
+        // corresponding operation. Therefore, the first belongs to the campaign setting operation,
+        // and the rest belong to the extension feed item operations.
+        printf(
+            "Removed a campaign extension setting with resource name: '%s'.%s",
+            $mutateOperationResponses[0]->getCampaignExtensionSettingResult()->getResourceName(),
+            PHP_EOL
+        );
+        for ($i = 1; $i < count($mutateOperationResponses); $i++) {
+            printf(
+                "Removed an extension feed item with resource name: '%s'.%s",
+                $mutateOperationResponses[$i]->getExtensionFeedItemResult()->getResourceName(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Creates a mutate operation for the sitelink campaign extension setting that will be removed.
+     *
+     * @param int $customerId the client customer ID
+     * @param int $campaignId the campaign ID
+     * @return MutateOperation the created mutate operation for the sitelink campaign extension
+     *     setting
+     */
+    private static function createSitelinkCampaignExtensionSettingMutateOperation(
+        int $customerId,
+        int $campaignId
+    ): MutateOperation {
+        // Creates the resource name of a campaign extension setting to remove.
+        $campaignExtensionSettingResourceName = ResourceNames::forCampaignExtensionSetting(
+            $customerId,
+            $campaignId,
+            ExtensionType::SITELINK
+        );
+
+        // Creates a campaign extension setting operation.
+        $campaignExtensionSettingOperation = new CampaignExtensionSettingOperation();
+        $campaignExtensionSettingOperation->setRemove($campaignExtensionSettingResourceName);
+
+        // Creates a mutate operation for the campaign extension setting operation.
+        return new MutateOperation([
+            'campaign_extension_setting_operation' => $campaignExtensionSettingOperation]);
```

Comment:
nit: I am not sure if this is PSR-12 compliant, we usually put `])` to a new line.

Reply:
I know you will ask, so I've checked before pushing this. :)
It's fine.
Parentheses will not be able to put like this, if the number of arguments is more than one.